common/data_format.hpp

```diff
@@ -35,18 +35,34 @@ namespace ILLIXR {
 		ullong dataset_time;
 	} imu_cam_type;
 
+	typedef struct {
+		Eigen::Matrix<double,3,1> w_hat;
+		Eigen::Matrix<double,3,1> a_hat;
+		Eigen::Matrix<double,3,1> w_hat2;
+		Eigen::Matrix<double,3,1> a_hat2;
+		Eigen::Matrix<double,13,1> pose;
+		time_type imu_time;
+	} imu_biases_type;
```

Comment:
These are not biases. They are actual IMU readings. Rename both the datatype and the variable name (here as well as in OV).